app/src/main/res/layout/fragment_home.xml

```diff
@@ -48,81 +48,77 @@
         android:background="@color/bottom_bar_shadow"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintBottom_toTopOf="@id/bottom_bar"/>
+        app:layout_constraintBottom_toTopOf="@id/bottom_bar" />
```

Comment:
nit: unnecessary ws change

However, there's no point in the churn to fix this given it's the only update so let's just land it as is.